src/ui/components/UserMenu/UserMenu.jsx

```diff
@@ -1,20 +1,22 @@
 import React from 'react';
 import { withRouter } from 'react-router';
 
-import Menu from 'material-ui/Menu';
-import MenuItem from 'material-ui/MenuItem';
+import { Menu, MenuItem, Divider } from 'material-ui';
```

Comment:
Wo wird der Divider verwendet? Ich habe den in den Komponenten nicht eingebunden da wir im Menü ja keine Unterteilungen haben. Schaden tuts aber sicher nicht.